XYZDrivers/src/main/java/com/xyzdrivers/controllers/ClaimsController.java

```diff
@@ -1,51 +1,46 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
+/**
+ * @file ClaimsController.java
+ * @author Nathan
+ * @created 30/10/17
+ * @modified 03/11/17
+ * @notes -
  */
+
 package com.xyzdrivers.controllers;
 
+import com.xyzdrivers.models.Claim;
+import com.xyzdrivers.services.InsertClaimService;
 import java.io.IOException;
-import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.time.LocalDate;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/**
- *
- * @author arthur
- */
 public class ClaimsController extends HttpServlet {
 
-    /**
-     * Processes requests for both HTTP <code>GET</code> and <code>POST</code>
-     * methods.
-     *
-     * @param request servlet request
-     * @param response servlet response
-     * @throws ServletException if a servlet-specific error occurs
-     * @throws IOException if an I/O error occurs
-     */
     protected void processRequest(HttpServletRequest request, HttpServletResponse response)
             throws ServletException, IOException {
-        
-        response.setContentType("text/html;charset=UTF-8");
-        try (PrintWriter out = response.getWriter()) {
-            
-            out.println("<!DOCTYPE html>");
-            out.println("<html>");
-            out.println("<head>");
-            out.println("<title>Servlet ClaimsController</title>");            
-            out.println("</head>");
-            out.println("<body>");
-            out.println("<h1>Servlet ClaimsController at " + request.getContextPath() + "</h1>");
-            out.println("</body>");
-            out.println("</html>");
-            
-                     
+
+        try {
+            LocalDate date = LocalDate.now();
```

Comment:
Is this insert code running on GET or POST? It should be post only

Reply:
Can move to doPost

Reply:
That's using the default method which overrides both doGet and doPost. Would it be worth me moving it out so that it's only in the doPost?

Reply:
Have moved the code to doPost.